.github/workflows/ci.yml

```
@@ -0,0 +1,77 @@
+name: Django Loci Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Python==${{ matrix.python-version }} | ${{ matrix.django-version }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7]
+        django-version: 
+          - django~=3.0
+          - django~=3.1
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install system packages
+      run: |
+        sudo apt update
+        sudo apt-get -qq -y install sqlite3 libsqlite3-dev libsqlite3-mod-spatialite gdal-bin
+
+    - name: Install python dependencies
+      run: |
+        pip install -r requirements-test.txt
```

Write this in single line only
```suggestion
      run: pip install -r requirements-test.txt
```